content/blog/hello-pancakes.md

```diff
@@ -1,8 +1,9 @@
 ---
 title: Dear gRPC
-author: April Nassi
-author-link: https://www.thisisnotapril.com/
 date: 2019-03-08
+author:
+  name: April Nassi
+  link: https://www.thisisnotapril.com/
```

Comment:
@thisisnotapril - the link doesn't seem to be valid.

Also, maybe you'd like your position to be added? If so, what would it be: "Open Source Strategist & Developer Relations Program Manager"? Or maybe just "Google"?

Reply:
Looks like HTTP works but not HTTPS. I've updated the link.